tests/src/RoutesAndParams.test.js

```diff
@@ -275,4 +267,209 @@ describe('RoutesAndParams /routes-and-params/inner-html', () => {
     expect(element).toBeTruthy();
   });
 
-});
\ No newline at end of file
+});
+
+describe('RoutesAndParams /routes-and-params/hrefs', () => {
```

Comment:
```suggestion
describe('RoutesAndParams /routes-and-params/hrefs spa', () => {
```

and move ssr to end of the last one as well to follow the pattern